gcs-triggered/src/main/java/com/byam/beam/examples/Options.java

```diff
@@ -0,0 +1,14 @@
+package com.byam.beam.examples;
+
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptions;
+import org.apache.beam.sdk.options.Validation.Required;
+import org.apache.beam.sdk.options.ValueProvider;
+
+public interface Options extends PipelineOptions {
+
+    @Description("Cloud Pub Sub Topic Name.")
+    @Required
+    ValueProvider<String> getTopic();
+    void setTopic(ValueProvider<String> value);
```

Comment:
I think it's ok to use `String` as an argument for setTopic.
What is the benefit of using `ValueProvider` here?

Reply:
It is useful for using Google Cloud Template. But i will change it to usual `string`. 
https://cloud.google.com/dataflow/docs/templates/creating-templates#using-staticvalueprovider